inspirehep/config.py

```diff
@@ -338,6 +338,13 @@
         'application/json': 'invenio_records_rest.serializers:json_v1_search',
         'application/vnd+inspire.ids+json': 'inspirehep.modules.api.v1.common_serializers:json_recids_response',
     },
+    'suggesters': {
+        'author': {
+            'completion': {
```

Comment:
` '_source': False ` will have to be added before the `completion` key in order to support source filtering IF https://github.com/inveniosoftware/invenio-records-rest/pull/226 gets merged.